pros/src/lib.rs

```diff
@@ -84,13 +84,11 @@ pub mod lvgl;
 pub mod time;
 pub mod usd;
 
-pub use async_trait::async_trait;
-
 pub type Result<T = ()> = core::result::Result<T, alloc::boxed::Box<dyn core::error::Error>>;
 
-use alloc::{boxed::Box, ffi::CString, format};
+use alloc::{ffi::CString, format};
 
-#[async_trait::async_trait]
+#[allow(async_fn_in_trait)]
```

Comment:
This is not the preferred way to do this. Instead of this just manually write out ``-> impl Future<Result>``.